dotnet/src/SemanticKernel.Skills/Skills.Document/FileSystem/LocalFileSystemConnector.cs

```diff
@@ -72,4 +72,10 @@ public Task<bool> FileExistsAsync(string filePath, CancellationToken cancellatio
     {
         return Task.FromResult(File.Exists(Environment.ExpandEnvironmentVariables(filePath)));
     }
+
+    /// <inheritdoc/>
+    public Task<string[]> RecursivelyListFilesUnderDirectoryAsync(string directoryPath, CancellationToken cancellationToken = default)
+    {
+        return Task.FromResult(Directory.GetFiles(directoryPath, "*.*", SearchOption.AllDirectories));
```

Comment:
It would be nice to allow the search pattern to be specified and have "*.*" as the default value.

Reply:
Will add